server/data/mappers/ciagPlanMapper.ts

```diff
@@ -0,0 +1,128 @@
+import type { InductionDto } from 'dto'
+import CiagPlan from '../ciagApi/interfaces/ciagPlan'
+import HopingToGetWorkValue from '../../enums/hopingToGetWorkValue'
+import TypeOfWorkExperienceValue from '../../enums/typeOfWorkExperienceValue'
+import WorkInterestsValue from '../../enums/workInterestsValue'
+import SkillsValue from '../../enums/skillsValue'
+import PersonalInterestsValue from '../../enums/personalInterestsValue'
+import InPrisonWorkValue from '../../enums/inPrisonWorkValue'
+import InPrisonEducationValue from '../../enums/inPrisonEducationValue'
+
+const toCiagPlan = (inductionDto: InductionDto): CiagPlan => {
+  return {
+    offenderId: inductionDto.prisonNumber,
+    desireToWork: inductionDto.workOnRelease.hopingToWork === HopingToGetWorkValue.YES,
+    hopingToGetWork: inductionDto.workOnRelease.hopingToWork,
+    reasonToNotGetWork: inductionDto.workOnRelease.notHopingToWorkReasons,
+    reasonToNotGetWorkOther: inductionDto.workOnRelease.notHopingToWorkOtherReason,
+    abilityToWork: inductionDto.workOnRelease.affectAbilityToWork,
+    abilityToWorkOther: inductionDto.workOnRelease.affectAbilityToWorkOther,
+
+    workExperience: toWorkExperience(inductionDto),
+    skillsAndInterests: toSkillsAndInterests(inductionDto),
+    qualificationsAndTraining: toQualificationsAndTraining(inductionDto),
+    inPrisonInterests: toInPrisonInterests(inductionDto),
+
+    createdBy: inductionDto.createdBy,
+    createdDateTime: inductionDto.createdAt.toISOString(),
+    modifiedBy: inductionDto.updatedBy,
+    modifiedDateTime: inductionDto.updatedAt.toISOString(),
+    prisonId: inductionDto.updatedAtPrison,
+    prisonName: undefined, // TODO - check this is a sound decision (not sure what else we might be able to do here)
```

Comment:
Is this used in any of the view templates? (I did a search and can't see where it is). Perhaps we could remove any references to it in a separate PR? Either way, I think it would be good to add a JIRA number here.

Reply:
Oh yeah; good point! That TODO was for me as part of this PR to double check `prisonName` is not being used, because we can't map it from our `InductionDto`. If it turns out the view or something needs it then we'll need to look it up somewhere/some how

Reply:
I have just checked, and yes, you are right, its not used in any view templates 👍 So I have removed my TODO comment

It is used in other mappings that maps from a `CiagPlan` to the various create and update REST API requests. However, our version of the CIAG API does not include `prisonName` for any operations (GET, POST, PUT), so I think you are right in that we can plan to remove `prisonName` from `CiagPlan` and related classes in a separate PR 👍

Reply:
Cool. It might be worth adding a sub-task or story for it 👍

Reply:
New story added for the removal of `prisonName` - RR-613